applications/wrapper.in

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
+import os, pwd, socket, requests, json, sys, logging
+from job_runner import LOG_URL
+
+
+def report(executable):
+    pw_entry = pwd.getpwuid(os.getuid())
+    user = pw_entry.pw_name
+
+    payload = {"user": user,
+                "cwd": os.getcwd(),
+                "application": executable,
+                "node": socket.gethostname(),
+                "komodo_release": os.getenv("KOMODO_RELEASE", "--------")
+    }
+
+    # Disabling proxies
+    proxies = {
+        "http": None,
+        "https": None,
+    }
+
+    requests.post(LOG_URL, timeout=3,
+                    headers={"Content-Type": "application/json"},
+                    data=json.dumps(payload), proxies=proxies)
+    
+if __name__ == "__main__":
+    executable = "${EXEC}"
+    try:
+        report(executable)
+    except Exception as e:
+        logging.warning("An error occured when logging application usage. "\
+            "Please contact the ERT-dev team at https://equinor.slack.com/messages/CDPJULWR4/")
```

Comment:
This is a tiny bit too internal for an open source project ;) I suggest you remove the second line.

Reply:
:+1: